ui/raidboss/data/05-shb/raid/e12s.js

```diff
@@ -146,10 +138,183 @@ const primalOutputStrings = {
   },
 };
 
+// Due to changes introduced in patch 5.2, overhead markers now have a random offset
+// added to their ID. This offset currently appears to be set per instance, so
+// we can determine what it is from the first overhead marker we see.
+// The first 1B marker in the encounter is the formless tankbuster, ID 004F.
+const firstHeadmarker = parseInt('00DA', 16);
+const getHeadmarkerId = (data, matches) => {
+  // If we naively just check !data.decOffset and leave it, it breaks if the first marker is 004F.
```

Comment:
```suggestion
  // If we naively just check !data.decOffset and leave it, it breaks if the first marker is 00DA.
```

For now, until we move this function into a utility file somewhere?